docs/extension.md

```diff
@@ -86,3 +90,92 @@ class ExtensionExample {
     }
 }
 ```
+
+### Access your Extension's service in Spring Boot
+In Spring Boot, your extension's service is _lazily_ initialized. It means that you must annotate your injected extension's service
```

Comment:
We could do it with https://www.baeldung.com/spring-boot-custom-auto-configuration too
But that would probably forces us to have a spring boot group extension module

Reply:
or maybe even simpler with https://www.baeldung.com/spring-componentscan-filter-type#filtertypeassignabletype ?

Reply:
Yeaaaah https://github.com/finos/symphony-bdk-java/pull/620/commits/839c08d7e024acbef38ccff5193781b465c6ac9f

Reply:
🪄 🐰 🎩